pom.xml

```diff
@@ -42,6 +42,16 @@
       <groupId>com.jayway.jsonpath</groupId>
       <artifactId>json-path</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.mashape.unirest</groupId>
+      <artifactId>unirest-java</artifactId>
+      <version>1.4.9</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
```

Comment:
Looks good but I think this can go too as we're using the built in one now.

Reply:
Oops, yep missed this

Reply:
Done